src/AutoMapper/Configuration/MapperConfigurationExpression.cs

```diff
@@ -26,8 +68,6 @@ public void CreateProfile(string profileName, Action<IProfileExpression> config)
 
         public IList<IObjectMapper> Mappers { get; }
 
-        public AdvancedConfiguration Advanced { get; } = new AdvancedConfiguration();
```

Comment:
Not exactly correct, but it seems reasonable to replace `Advanced` with `Internal()`. Easier to upgrade. If some setting is too important for that, we could move it in the public interface.